flask_server.py

```diff
@@ -67,22 +71,28 @@ def index():
     if request.method == "POST" and is_user_logged_in:
         amount_to_scrape = int(request.form["amountToScrape"]) if request.form["amountToScrape"].isdecimal() else -1
         mode, ids_mode = request.form["mode_dropdown"], request.form["ids_dropdown"]
+        logger.info(f"Starting scraping in mode: {mode}, ids source: {ids_mode}")
         base64_media = start_scrape(settings.get(), folder_path, amount_to_scrape, mode, ids_mode)
+        logger.info("Finished scraping")
```

Comment:
Move the logger.info inside start_scrape, this way this also works for runner.py